usaspending_api/accounts/v2/filters/account_download.py

```diff
@@ -437,6 +438,23 @@ def award_financial_derivations(derived_fields):
         output_field=TextField(),
     )
 
+    derived_fields["prime_award_summary_recipient_cd_original"] = congressional_district_display_name(
+        "award__latest_transaction_search__recipient_location_state_code",
+        "award__latest_transaction_search__recipient_location_congressional_code",
```

Comment:
Award Search and the latest transaction (from Transaction Search) should be populated with the same CD data. I'd say you can skip the extra join to Transaction Search to pull these fields. I see a similar pattern throughout the code review.

Reply:
This is how it was before and follows how all the other fields are populated. Are you sure we want to make this a requirement under this story since this is consistent with all the other fields

Reply:
Per slack conversation, I'm marking this comment as resolved.

> Current implementation is probably good for now. If other fields are already joining, it won’t provide much (if any) of a performance improvement to refactor